admin-dev/themes/new-theme/js/pages/order/create.js

```diff
@@ -61,10 +61,22 @@ function refreshCart() {
   orderPageManager.refreshCart();
 }
 
+/**
+ * proxy to allow other scripts within the Create Order page to refresh cart
+ */
+function addCreatedCartRuleToCart(cartRuleId) {
+  if (orderPageManager === null) {
+    console.log('Error: Could not refresh created cart rule to the current cart as orderPageManager is null');
+    return;
+  }
+  orderPageManager.addCreatedCartRuleToCart(cartRuleId);
```

Comment:
```suggestion
  orderPageManager.addCreatedCartRuleToCart(cartRuleId);
  orderPageManager.refreshCart();
```

Reply:
Why not applying this suggestion @sowbiba ?

Reply:
addCreatedCartRuleToCart already perform a refreshCart